uaclient/cli.py

```diff
@@ -273,17 +274,6 @@ def action_attach(args, cfg):
         ua_status.MESSAGE_ATTACH_SUCCESS_TMPL.format(
             contract_name=contract_name))
 
-    # sort the list, so that we get consistent attach behaviour regardless of
```

Comment:
Dropped this because individual services will be handling enableByDefault during process_contract_deltas.